domain/consensus/processes/transactionvalidator/transactionvalidator_test.go

```diff
@@ -0,0 +1,256 @@
+
+		txInput := externalapi.DomainTransactionInput{
+			PreviousOutpoint: prevOutPoint,
+			SignatureScript:  []byte{},
+			Sequence:         constants.MaxTxInSequenceNum,
+			UTXOEntry: utxo.NewUTXOEntry(
+				100_000_000, // 1 KAS
+				scriptPublicKey,
+				true,
+				uint64(5)),
+		}
+		txInputWithMaxSequence := externalapi.DomainTransactionInput{
+			PreviousOutpoint: prevOutPoint,
+			SignatureScript:  []byte{},
+			Sequence:         constants.SequenceLockTimeIsSeconds,
+			UTXOEntry: utxo.NewUTXOEntry(
+				100000000, // 1 KAS
+				scriptPublicKey,
+				true,
+				uint64(5)),
+		}
+		txInputWithLargeEntry := externalapi.DomainTransactionInput{
+			PreviousOutpoint: prevOutPoint,
+			SignatureScript:  []byte{},
+			Sequence:         constants.MaxTxInSequenceNum,
+			UTXOEntry: utxo.NewUTXOEntry(
+				constants.MaxSompi,
+				scriptPublicKey,
+				true,
+				uint64(5)),
+		}
+
+		txOut := externalapi.DomainTransactionOutput{
+			Value:           100000000, // 1 KAS
+			ScriptPublicKey: scriptPublicKey,
+		}
+		txOutBigValue := externalapi.DomainTransactionOutput{
+			Value:           200_000_000, // 2 KAS
+			ScriptPublicKey: scriptPublicKey,
+		}
+
+		validTx := externalapi.DomainTransaction{
+			Version:      constants.MaxTransactionVersion,
+			Inputs:       []*externalapi.DomainTransactionInput{&txInputWithMaxSequence},
+			Outputs:      []*externalapi.DomainTransactionOutput{&txOut},
+			SubnetworkID: subnetworks.SubnetworkIDRegistry,
+			Gas:          0,
+			LockTime:     0}
+		txForCoinbaseMaturityCheck := externalapi.DomainTransaction{
+			Version:      constants.MaxTransactionVersion,
+			Inputs:       []*externalapi.DomainTransactionInput{&txInput},
+			Outputs:      []*externalapi.DomainTransactionOutput{&txOut},
+			SubnetworkID: subnetworks.SubnetworkIDRegistry,
+			Gas:          0,
+			LockTime:     0}
+		txForInputsAmountsCheck := externalapi.DomainTransaction{
+			Version:      constants.MaxTransactionVersion,
+			Inputs:       []*externalapi.DomainTransactionInput{&txInput, &txInputWithLargeEntry},
+			Outputs:      []*externalapi.DomainTransactionOutput{&txOut},
+			SubnetworkID: subnetworks.SubnetworkIDRegistry,
+			Gas:          0,
+			LockTime:     0}
+		txForOutputsAmountsCheck := externalapi.DomainTransaction{
+			Version:      constants.MaxTransactionVersion,
+			Inputs:       []*externalapi.DomainTransactionInput{&txInput},
+			Outputs:      []*externalapi.DomainTransactionOutput{&txOutBigValue},
+			SubnetworkID: subnetworks.SubnetworkIDRegistry,
+			Gas:          0,
+			LockTime:     0}
+		txForSequenceLockCheck := externalapi.DomainTransaction{
+			Version:      constants.MaxTransactionVersion,
+			Inputs:       []*externalapi.DomainTransactionInput{&txInput},
+			Outputs:      []*externalapi.DomainTransactionOutput{&txOut},
+			SubnetworkID: subnetworks.SubnetworkIDRegistry,
+			Gas:          0,
+			LockTime:     0}
+		txForScriptsCheck := externalapi.DomainTransaction{
```

Comment:
Can you rename this to explain what's special about the transaction, and not for what test it's used?